base/tools/startshell/startshell.c

```diff
@@ -28,8 +28,8 @@ inst_start_shell (char *tty_tv)
     setsid ();
     fd_ii = open (tty_tv, O_RDWR);
     ioctl (fd_ii, TIOCSCTTY, (void *)1);
-    dup (fd_ii);
-    dup (fd_ii);
+    if (dup (fd_ii));
+    if (dup (fd_ii));
```

Comment:
to be honest I do not get this code. I think we need to document it. Why we need duping? what happen if it failed?